docs/resources/application_access_grant_approval.md

```diff
@@ -0,0 +1,39 @@
+# axual_application_access_grant_approval (Resource)
+
+Application Access Grant Approval: Approve access to a stream
+
+## Limitations
+- Approval cannot be edited
+- Revoke grant by destroying application_access_grant_approval
+- Read more: https://docs.axual.io/axual/2023.1/self-service/topic-authorizations.html#contents
+
+## Usage
+- To approve a grant create an application_access_grant_approval with the grant id.
+- To revoke a grant, delete the application_access_grant_approval
+- To revoke an auto approved grant
+- - Create an approval to import it. 
+- - Apply the terraform state.
+- - Then delete that approval.
```

Comment:
Auto approved grants are not meant to be revoked. When stream configs are removed, the corresponding grants should be automatically removed.

Reply:
The current API does not make that distinction. All of them can be revoked. And existance of a grant prevents deleting of stream configs.

Reply:
Okay then the current implementation is fine 👍